src/cow-react/modules/swap/pure/EthFlow/EthFlowStepper/steps/Progress2.tsx

```diff
@@ -19,7 +19,7 @@ export function Progress2({ order, refund, cancelation }: EthFlowStepperProps) {
       }
     }
 
-    if (refundTx || cancelationTx) {
+    if (refundTx || cancellationTx) {
```

Comment:
Are you british?

> Canceled and canceling are more common in the US, while cancelled and cancelling are more common in British English

https://www.merriam-webster.com/words-at-play/canceled-or-cancelled

Reply:
Better, Canadian!